scripts/utils/sign_and_send.js

```diff
@@ -7,6 +7,7 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
   const axios = require("axios")
   const { getSafeCompatibleSignature, estimateGas } = require("../utils/internals")(web3, artifacts)
   const { ZERO_ADDRESS } = require("./constants")
+  const baseGas = 0
```

Comment:
Very much nitpick, but I think that the name is too simple if we want to move this constant to the preamble of the script. I'd consider leaving the constant where it was and duplicate it the next time you use it.

Reply:
Okay, I made the name more descriptive. I really would like to avoid two different variables, as this could cause discrepancies in the transactions later on.